pkg/v1/tkg/web/node-server/src/routes/api/mockService.js

```diff
@@ -69,6 +69,7 @@ router.get(`${ENDPOINT}/features`, (req, res) => {
             "import": "true",
             "vsphereIPv6": "true",
             "standalone-cluster-mode": "false",
+            "clusterClassSupport": "true"
```

Comment:
I think most of the flags use the hyphenated case, so cluster-class-support might be more consistent. However, I have no strong opinion

Reply:
+1

Reply:
renamed